### PR TITLE
Docs: fix missed properties in document of `automation`, `network`, `fluidrelay server`, etc.

### PR DESCRIPTION
**Pull request by wuxu92**
fix document typos or missed properties of some RPs.

### CODE REVIEW

File: website/docs/r/automation_module.html.markdown [hunk 1]
```diff
@@ -46,14 +46,24 @@ The following arguments are supported:
 
 * `automation_account_name` - (Required) The name of the automation account in which the Module is created. Changing this forces a new resource to be created.
 
-* `module_link` - (Required) The published Module link.
+* `module_link` - (Required) The published Module link `module_link` block defined as blow.
```
Comment (stephybun) [inline website/docs/r/automation_module.html.markdown:49]:
```suggestion
* `module_link` - (Required) A `module_link` block as defined below.
```

File: website/docs/r/automation_module.html.markdown [hunk 2]
```diff
@@ -46,14 +46,24 @@ The following arguments are supported:
...
 
 ---
 
 The `module_link` block supports the following:
 
 * `uri` - (Required) The URI of the module content (zip or nupkg).
 
+* `hash` - (Optional) A `hash` block defined as below.
```
Comment (stephybun) [inline website/docs/r/automation_module.html.markdown:57]:
```suggestion
* `hash` - (Optional) A `hash` block as defined below.
```

**Review by stephybun — CHANGES_REQUESTED** [review]
Thanks for this @wuxu92, left some suggestions in line but once those are fixes this should be good to go.

**Review by katbyte — APPROVED** [review]
LGTM ⚒️

**Pull request merged**